tests/ethernet/eth.py

```diff
@@ -0,0 +1,23 @@
+import utils
```

Comment:
Please move all common functions to utils itself. If you think it belongs to a specific separate module, feel free to create eth.py inside `utils/`